src/main/java/twistedgate/immersiveposts/client/ClientProxy.java

```diff
@@ -62,11 +63,11 @@ public void setupManualPage(){
 			int page=1;
 			for(int i=0;i<index0.length;i++){
 				index0[i][0]=new StringTextComponent(Integer.toString(page++));
-				index0[i][1]=new StringTextComponent("index.page_0_entry."+Integer.toString(i+1));
+				index0[i][1]=new TranslationTextComponent("index.page_0_entry."+Integer.toString(i+1));
```

Comment:
Im guessing this one and the other dont work anymore like they used to

Reply:
Confused by what you mean, this didn't work at all in the currently published version because you need the Translation component for it to key into the lang file. The String component just uses the literal string you supply.

Reply:
The thing is, it *did* work.. otherwise it wouldnt be there :D
hence my confusion

Reply:
Weird, I think for as long as I've been modding, you've needed a Translation component for that to work. The name has changed over the years, but I think since at least 1.7, that's been the case. Must've been a glitch or something! Either way, it works now.

Reply:
Did some digging, for this specific case, it pertains to an API change in Immersive Engineering for 1.16. Didn't dig too far, but it appears that they changed from just taking Strings (that they must have implicitly resolved using a I18n lookup, not sure) to taking ITextComponents directly, allowing more direct control from API-implementers. So it looks like you just selected StringTextComponent since that was the most basic one, where you actually wanted the TranslationTextComponent.

Reply:
Huh, might be one of those "i was lazy" cases lol